test/server/organelles/polls.js

```diff
@@ -0,0 +1,204 @@
+const async = require('async')
+const mongoose = require('mongoose')
+
+const User = require('../../../server/models/user')
+const Poll = require('../../../server/models/poll')
+const PollsOrganelle = require('../../../server/organelles/polls')
+
+describe('organelles/polls', function () {
+  const ObjectId = mongoose.Types.ObjectId
+  const user1Id = ObjectId()
+  const user2Id = ObjectId()
+  const pollId = ObjectId()
+
+  before(done => {
+    test.startServer(err => {
+      if (err) return done(err)
+      // insert test data
+      async.series([
+        next => {
+          User
+            .insertMany([{
+              _id: user1Id,
+              email: 'aivo@devlabs.bg',
+              password: 'nd2ibd57GJlAjOpS',
+              firstname: 'Ivaylo',
+              lastname: 'Atanasov',
+              rate: 50
+            }, {
+              _id: user2Id,
+              email: 'veselin@devlabs.bg',
+              password: 'nd2ibd57GJlAjOpS',
+              firstname: 'Veselin',
+              lastname: 'Bratanov',
+              rate: 100
+            }])
+            .catch(err => next(err))
+            .then(users => next())
+        },
+        next => {
+          Poll
+            .insertMany([{
+              _id: pollId,
+              userId: user1Id,
+              rate: 60,
+              completedAt: null,
+              approved: null,
+              votes: []
+            }, {
+              userId: user2Id,
+              rate: 90,
+              completedAt: null,
+              approved: null,
+              votes: []
+            }])
+            .catch(err => next(err))
+            .then(polls => next())
+        }
+      ], (err, result) => {
+        if (err) return done(err)
+        return done(null)
+      })
+    })
+  })
+
+  after(done => {
+    async.series([
+      next => {
+        mongoose.connection.db.dropDatabase(next)
+      },
+      test.stopServer
+    ], done)
+  })
```

Comment:
I'll be copying this behaviour on my branch (https://github.com/dev-labs-bg/rate-me-v2/pull/4/commits/7576a455d69131cdcca1bb793d4fe7190fcff4c8)
We can make them the same later on.